### ISSUE TITLE
TechTypes is seemingly missing a value for the Shield Battery's Recharge Shields

### ISSUE THREAD

**Issue by dgant** [bwapi/bwapi#725]
Right now there's no way to tell a Shield Battery to use its Recharge Shields ability. I'm assuming that under the hood this would be using a TechType -- that assumption could very well be mistaken.

There is a workaround -- you can order a unit you want healed to right-click the Battery, but there's no way to explicitly tell the Shield Battery to use its ability.

**Comment by dgant** [comment]
Turns out the correct way is using an order, not using a tech. Closed.

**Comment by dgant** [comment]
Quoting @tscmoo : "In the game engine itself there is no tech type for it" -- So, not that, but perhaps a helper ability rechargeShields() would be fitting. "Won't fix" is also reasonable, but might merit a note in the documentation for useTech.

**Comment by davechurchill** [comment]
You can right click the shield battery with a unit to recharge its shields, no?

**Comment by dgant** [comment]
Yep -- see above comment. But the workaround has confused more than a few developers. Again, it's a pretty minor concern either "Closed/Won't fix" or a hint in the documentation are both reasonable resolutions.

**Comment by heinermann** [comment]
I think we could provide a `unitA.rechargeShields(unitB)` command which performs the proper action depending on which unit is the shield battery. The shield battery also has a recharge command that you can use on a unit, but the common means of use is for a unit to right click the battery.
